packages/gatsby/src/utils/reporter/errors.js

```diff
@@ -0,0 +1,67 @@
+// @flow
```

Comment:
This should be in gatsby-cli? Or is this a duplicate?

Reply:
DUP!